sdks/java/core/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
   shadow library.java.junit
   shadow "org.tukaani:xz:1.5"
   shadowTest project(":model:fn-execution").sourceSets.test.output
+  shadowTest project(path: ":runners:core-construction-java", configuration: "shadow")
```

Comment:
What introduced this edge?

This also can't be represented in maven, and right now that worries me (though hopefully less soon)

Reply:
This isn't needed anymore. Thanks for pointing it out.